### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by tweirtx**

### CODE REVIEW

File: test/api_status_app_version_test.dart [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import 'package:openapi/api.dart';
+import 'package:tba_api_client/api.dart';
```
Comment (fletch3555) [inline test/api_status_app_version_test.dart:1]: I feel like these namespace changes should be handled by the generator....

File: test/api_status_app_version_test.dart [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 import 'package:test/test.dart';
 
 // tests for APIStatusAppVersion
 void main() {
-    var instance = new APIStatusAppVersion();
```
Comment (fletch3555) [inline test/api_status_app_version_test.dart:6]: Is dart whitespace-sensitive like python, or were you just tidying up a bit with these?

File: test/api_status_test.dart [hunk 3]
```diff
@@ -25,7 +25,7 @@ void main() {
     });
 
     // An array of strings containing event keys of any active events that are no longer updating.
-    // List<String> downEvents (default value: [])
+    // List<String> downEvents (default value: const [])
```
Comment (fletch3555) [inline test/api_status_test.dart:28]: Are the comment changes necessary?

**Review by fletch3555 — COMMENTED** [review]
Seems like most of the fixes provided here are more pedantry than anything else.  Though I don't really know Dart, so it's possible my questions are silly

File: test/api_status_app_version_test.dart (hunk 1 above)
Comment (tweirtx) [reply test/api_status_app_version_test.dart:1]: They should, and yet here we are

File: test/api_status_app_version_test.dart (hunk 2 above)
Comment (tweirtx) [reply test/api_status_app_version_test.dart:6]: When I dartfmted the project, it made those changes

File: test/api_status_test.dart (hunk 3 above)
Comment (tweirtx) [reply test/api_status_test.dart:28]: No, that was a find and replace I think

File: test/api_status_app_version_test.dart (hunk 2 above)
Comment (fletch3555) [reply test/api_status_app_version_test.dart:6]: ah okay, so just a formatter.  Makes sense.

File: test/api_status_app_version_test.dart (hunk 1 above)
Comment (fletch3555) [reply test/api_status_app_version_test.dart:1]: haha fair enough

**Comment by aidan-mundy** [comment]
Is this going to be merged in? It seems to fix the compile time errors I have with the existing version, and it passes Travis

**Comment by fletch3555** [comment]
This repo is generated, so I'm hesitant because it may get overwritten.

**Comment by aidan-mundy** [comment]
As-generated, dart programs using the API wont compile due to some type mismatching stuff

**Comment by fletch3555** [comment]
I understand that.  That just tells me that I need to figure out how to fix the generator, not merge this.

**Comment by aidan-mundy** [comment]
Ahh, understood

**Pull request closed without merging**